servers/partner-ci/common.yaml

```diff
@@ -341,8 +342,7 @@
                 ${SNAPSHOTS_URL/SNAPSHOTS_ID/$SNAPSHOTS_ID}
                 snapshots_id=$(grep "^CUSTOM_VERSION=" < snapshots.params | cut -d= -f2 | cut -d'#' -f2)
                 echo "snapshot_id_number=" $snapshots_id
-                python plugin_test/fuel-qa/utils/jenkins/conv_snapshot_file.py
-
+                python "${CONV_SNAP_PATH}/conv_snapshot_file.py"
```

Comment:
Let's define full path to the util:
`CONV_SNAP_PATH=utils/jenkins/conv_snapshot_file.py`